Template.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net5</TargetFramework>
+		<PackageType>Template</PackageType>
+		<PackageVersion>1.0.0</PackageVersion>
+		<PackageId>GmodNET.Templates</PackageId>
+		<Title>GmodDotNet Templates</Title>
+		<Authors>GmodNET</Authors>
+		<Description>Project Templates for GmodDotNet module</Description>
+		<PackageTags>dotnet-new;templates</PackageTags>
+		<IncludeContentInPack>true</IncludeContentInPack>
+		<IncludeBuildOutput>false</IncludeBuildOutput>
+		<ContentTargetFolders>content</ContentTargetFolders>
+	</PropertyGroup>
+	<ItemGroup>
+		<Content Include="ProjectTemplates\**\*" Exclude="ProjectTemplates\**\bin\**;ProjectTemplates\**\obj\**" />
```

Comment:
Replace wildcards with explicit list of files. This will prevent unwanted objects from getting into the package.

Reply:
@GlebChili, but it's for easier way adding new templates

Reply:
Still we need list them explicitly.

Reply:
Resolved